### PR TITLE
Add draco glTF compression support

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: modules/core/src/scenegraph/gltf/gltf-instantiator.js [hunk 1]
```diff
@@ -155,15 +155,22 @@ export default class GLTFInstantiator {
     return loadedAttributes;
   }
 
-  createBuffer(bufferView, target) {
+  createBuffer(attribute, target) {
+    if (!attribute.bufferView) {
+      // Draco decoded files do not have a bufferView
```
Comment (ibgreen) [inline modules/core/src/scenegraph/gltf/gltf-instantiator.js]: Interesting point. Maybe we should add new bufferViews, after decoding.

I removed the extension after decompression with the intention that the app wouldn't be able to tell that it was ever there, but here we are, adding special cases to handle it...

File: modules/core/src/scenegraph/gltf/gltf-instantiator.js [hunk 2]
```diff
@@ -155,15 +155,22 @@ export default class GLTFInstantiator {
...
     if (!bufferView.lumaBuffers) {
       bufferView.lumaBuffers = {};
     }
 
     if (!bufferView.lumaBuffers[target]) {
       bufferView.lumaBuffers[target] = new Buffer(this.gl, {
         id: `from-${bufferView.id}`,
-        data: bufferView.data,
+        // Draco decoded files have attribute.value
+        data: bufferView.data || attribute.value,
```
Comment (ibgreen) [inline modules/core/src/scenegraph/gltf/gltf-instantiator.js]: Yep, seems like this was returned by DracoDecoder, should remap this to fit with glTF model.

File: modules/core/src/scenegraph/gltf/gltf-instantiator.js (hunk 1 above)
Comment (georgios-uber) [reply modules/core/src/scenegraph/gltf/gltf-instantiator.js]: Maybe.

File: modules/core/src/scenegraph/gltf/gltf-instantiator.js (hunk 2 above)
Comment (georgios-uber) [reply modules/core/src/scenegraph/gltf/gltf-instantiator.js]: Yes please.

File: examples/gltf/app.js [hunk 3]
```diff
@@ -51,8 +52,12 @@ async function loadGLTF(urlOrPromise, gl, options = DEFAULT_OPTIONS) {
 
   const data = await promise;
 
-  const gltfParser = new GLTFParser({uri: urlOrPromise});
-  const gltf = await gltfParser.parseAsync(data);
+  const gltfParser = new GLTFParser();
+  const gltf = await gltfParser.parse(data, {
+    uri: urlOrPromise,
```
[the remark this answers is not in the document]
Comment (georgios-uber) [reply examples/gltf/app.js]: No, but maybe it should.

**Pull request merged**